src/libopensc/card-epass2003.c

```diff
@@ -3050,7 +3050,10 @@ epass2003_erase_card(struct sc_card *card)
 		0x31,0x32,0x33,0x34, 0x35,0x36,0x37,0x38,
 		0x31,0x32,0x33,0x34, 0x35,0x36,0x37,0x38,
 	};
-	static const unsigned char magic_pin[16] = "1234567812345678";
+	static const unsigned char magic_pin[16] = {
+		0x31,0x32,0x33,0x34, 0x35,0x36,0x37,0x38,
+		0x31,0x32,0x33,0x34, 0x35,0x36,0x37,0x38
+	};
```

Comment:
the only nit would be to make the clang format happy:
```suggestion
			0x31, 0x32, 0x33, 0x34, 0x35, 0x36, 0x37, 0x38,
			0x31, 0x32, 0x33, 0x34, 0x35, 0x36, 0x37, 0x38,
	};
```